src/sql/azdata.proposed.d.ts

```diff
@@ -541,13 +541,20 @@ declare module 'azdata' {
 
 		export type DialogWidth = 'narrow' | 'medium' | 'wide' | number;
 
+		export enum DialogStyle {
+			Normal = 0,
+			Flyout = 1,
+			Callout = 2
+		}
+
 		/**
 		 * Create a dialog with the given title
 		 * @param title The title of the dialog, displayed at the top
 		 * @param dialogName the name of the dialog
 		 * @param width width of the dialog, default is 'wide'
+		 * @param dialogStyle Enum representing choice of dialog style: flyout or callout.
```

Comment:
please document the default value: Flyout and implement it

Reply:
Done